tooling/prometheus-rules/main.go

```diff
@@ -72,11 +68,62 @@ func main() {
 	if err := o.complete(flag.Args()); err != nil {
 		logrus.WithError(err).Fatal("could not complete options")
 	}
-	if err := generate(o.rules, o.output); err != nil {
+	b, _ := os.ReadFile("testing-prometheusRule_test.yaml")
```

Comment:
a) why is this hard-coded?
b) why is the error ignored?

Reply:
Sorry, this PR should be a draft. going to change that. It's simply not ready